Day24/GlobalAssignment.scilla

```diff
@@ -0,0 +1,50 @@
+(********* Global Assignment *************)
+
+
+scilla_version 0
+
+library Miracle
+let not_owner_code  = Uint32 1
+
+contract Miracle
+(oracle: ByStr20)
+
+field currentquestionId : Int32 = Int32 0
+
+
+field finalOutput : Map Int32 String = Emp Int32 String
+field userVsAnswer: Map ByStr20 String = Emp ByStr20 String
+
+
+transition addQuestion (description : String)
+  
+
+end
+
+(******(GLobal Assignment Part-4, Season 3)*****)
+transition resultDeclaration (questionid: Int32, res: String)
+is_owner = builtin eq oracle _sender;
+match is_owner with
+  | False =>
+    e = {_eventname : "NotaMatch"; code : not_owner_code};
+    event e
+  | True =>
+    finalOutput[questionid] := res;
+    e = {_eventname : "Response"; outPut : res};
+    event e
+  end
+end
+(******(GLobal Assignment Part-4, Season 3)*****)
+
+transition userAnswer (questionid: Int32, usrRes: String)
+accept;
+answer = usrRes;
+userVsAnswer[_sender]:= answer
+
+
+end
+
+transition redeem (questionid: Int32)
```

Comment:
Implementation is missing. 
Let us know once implemented.

Reply:
implemented again can you check